modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -87,80 +87,295 @@ $ docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig registr
 
 This allows your `kubeconfig` file to be consumed from inside the running container.
 
-[id="cnf-performing-end-to-end-tests-running-the-latency_tests"]
-=== Running the latency tests
-In {product-title} {product-version}, you can also run latency tests from the CNF-test container. The latency test allows you to set a latency limit so that you can determine performance, throughput, and latency.
+[WARNING]
+====
+You must run the latency tests in Discovery mode. The latency tests can change the configuration of your cluster if you do not run in Discovery mode.
+====
 
-The latency test runs the `oslat` tool, which is an open source program to detect OS level latency. For more information, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/5315541[How to measure OS and hardware latency on isolated CPUs?].
+In {product-title} {product-version}, you can also run latency tests from the CNF-test container. The latency test allows you to validate the node tuning is sufficient for your workload.
```

Comment:
s/he latency test allows you to validate the node/the latency test allows you to validate that the node